imagedephi/redact/__init__.py

```diff
@@ -0,0 +1 @@
+from .redact import iter_image_files, redact_images, show_redaction_plan  # noqa
```

Comment:
You add [an `__all__` in this file](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) so that the `# noqa` isn't necessary.

Alternatively, you could make this file blank, and refactor imports to reach into the sub-module files.

Reply:
Or at least use `#noqa: ERRCODE`